luigi_bigquery/task.py

```diff
@@ -119,7 +119,7 @@ def requires(self):
         return DatasetTask(self.dataset())
 
     def output(self):
-        return TableTarget(self.dataset(), self.table())
+        return TableTarget(self.dataset(), self.table(), None, False, None, self.append())
```

Comment:
@abronte I think it's not necessary to add `append()` method and rewrite `self.write_disposition`.
Change this line to

```python
return TableTarget(self.dataset(), self.table(), append=(self.write_disposition == bigquery.JOB_WRITE_APPEND))
```

is enough. If you do so, just use `QueryTable` like this.

```
class MyQueryTableStep1(luigi_bigquery.QueryTable):
    write_disposition == bigquery.JOB_WRITE_APPEND

    def dataset(self):
        return 'luigi'

    def table(self):
        return 'tmp'

    def query(self):
        return "SELECT count(*) cnt FROM [publicdata:samples.github_nested]"
```